cms/settings.py

```diff
@@ -79,8 +80,10 @@
         "APP_DIRS": True,
         "OPTIONS": {
             "context_processors": {
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
                 "django.contrib.auth.context_processors.auth",
-                "django.template.context_processors.static",
```

Comment:
Is this now included by default or something?

Reply:
Yeah